content/english/facts/soy-is-bad-for-you.md

```diff
@@ -0,0 +1,223 @@
+> 4. "[T]o have the *beneficial* effects of soy isoflavones, intake should be at least of 60-100 mg per day" see *Soy, Soy Foods and Their Role in Vegetarian Diets - 2018* below)
+> 5. Speculation: It is reasonable to assume a similar dosage is required to see any negative effects.  
+> 6. It is _unlikely_ that you will ingest a high enough dosage to affect your health as part of a balanced diet provided you do not have any special circumstances (e.g. hypothyroidism, infancy, etc.) (see *Soy, Soy Foods and Their Role in Vegetarian Diets - 2018* below).  
+> 7. All articles on the topic caveat their conclusions with: 'more research is needed'.  
+> 8. Ensure that you vary your protein sources and consult your dietitian for special circumstances (e.g. hypothyroidism, infancy, etc.) see *Soy, Soy Foods and Their Role in Vegetarian Diets - 2018* below.  
+>  
+> \- TalkVeganToMe's layman's understanding from surveying the material available online
+
+Because of the complexity of this topic (and lack of definite declarations of safety from major organisations and articles not associated with the soy industry) we have opted to quote some of the most common sources presenting evidence for and against the risks of soy. This does not follow our normal pattern of debunking but does allow you to be well informed on the topic and arrive at your own conclusions.
+
+# Studies
+
+There is an *huge* amount of information available on the safety of soy, more than 2000 soy-related peer-reviewed articles are published annually [Soy and Health Update - Nutrients 2016](http://web.archive.org/web/20190923193109/https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5188409/).   
+Those which point to negative health effects of soy usually have extremely small sample sizes, and extremely high dosages (see *Showing Risk* below).   
+Those which suggest that there is no evidence of negative outcomes caveat these conclusions with the "a [lack] of long term data on soy consumption in Western countries". 
+
+We have summarised each below and (where possible linked to the original complete articles) for your convenience.
+
+## Suggesting Risk/Impact 
+
+### Cancer
+
+#### Dietary Soy and Increased Risk of Bladder Cancer - 2002
+> The association between soyfood consumption and subsequent bladdercancer risk was investigated in a population-based cohort study, the Singapore Chinese Health Study. As of December 31, 2000, 329,848 person-years of follow-up were accrued. Sixty-one histologically confirmed incident bladder cancer cases were identified. Information on soyfood consumption at baseline was obtained through in-person interviews using a validated dietary questionnaire. [...]  
+> High intake of soyfood was statistically significantly related to an elevated risk of bladder cancer.  
+> - [Dietary Soy and Increased Risk of Bladder Cancer - 2002](https://web.archive.org/web/20191009182415/https://cebp.aacrjournals.org/content/11/12/1674.long)
```

Comment:
Yes... BUT.... note at the end:

"We caution that our novel finding of a positive association between soy and bladder cancer risk is based on a relatively short period of follow-up and a modest number of cancer cases. If this provocative observation is confirmed by others..."

Rather the opposite has been true.

See: Soy Isoflavone Intake and Bladder Cancer Risk in Japan: From the Takayama Study